src/views/ResLife/components/RAView/components/RD-OnCall/index.jsx

```diff
@@ -14,12 +14,14 @@ import {
 import { styled } from '@mui/material/styles';
 import ScottieMascot from 'views/ResLife/ScottieMascot.png';
 import { getRDOnCall } from 'services/residentLife/RD_OnCall';
+import { formatPhoneNumber } from 'views/ResLife/utils/formatPhoneNumber/formatPhoneNumber';
 
 const COLOR_80808026_1X1 =
   'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNsUAMAASwAqHb28sMAAAAASUVORK5CYII=';
 
-const RD_PHONE_NUMBER = 'RD Phone number here';
+const RD_PHONE_NUMBER = '(978)-500-7259';
```

Comment:
Is this truly a constant? Maybe you can add a feature on the backlog to have this stored in the database and managed by the RDs?